### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by artyom123**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        extensions: ['*', '.js', '.jsx'],
+    },
+    plugins: [
+        new HtmlWebPackPlugin({
+            template: `${src}${indexHtml}`,
+            filename: `.${indexHtml}`,
+        }),
+        new MiniCssExtractPlugin({
+            filename: PROD ? '[name].css' : '[name].[hash].css',
+            chunkFilename: PROD ? '[id].css' : '[id].[hash].css',
```
Comment (Vladlan) [inline webpack.config.js]: Our production (dist) files should be hashed (and also js files).

File: package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "learn-react",
+  "version": "1.0.0",
+  "description": "Learn react",
+  "main": "index.js",
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --open",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "build:dev": "cross-env NODE_ENV=development webpack",
```
Comment (Vladlan) [inline package.json:9]: For what reason we need build.dev command?

**Review by Vladlan — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (artyom123) [reply package.json:9]: 1)	I want to see files and use source-map for easy debugging in build mode
2)	I don’t want to minify CSS and Html

File: webpack.config.js (hunk 1 above)
Comment (artyom123) [reply webpack.config.js]: Fix

**Pull request closed without merging**